client/src/pages/Checkout.js

```diff
@@ -7,20 +7,29 @@ import {
   Card,
   CardMedia,
   CardContent,
+  Button,
 } from "@material-ui/core";
 import { AddCircle, RemoveCircle } from "@material-ui/icons";
+
 import PaymentForm from "components/PaymentForm";
 import { makeStyles } from "@material-ui/core/styles";
 import { Context as MealContext } from "contexts/MealContext";
+import { Context as UserContext } from "contexts/AuthContext";
```

Comment:
we should keep it as import {Context as AuthContext}  (This was my mistake as i was first person to rename it UserContext)